library/threading.c

```diff
@@ -29,6 +37,23 @@
 
 #include "mbedtls/threading.h"
 
+#if !defined(_WIN32) && (defined(unix) || \
```

Comment:
I think this should be guarded by `HAVE_DATE_TIME` and `! MBEDTLS_PLATFORM_GMTIME_ALT`.